libgit2-sys/lib.rs

```diff
@@ -3559,6 +3581,7 @@ extern "C" {
         ancestor: *const git_oid,
     ) -> c_int;
 
+    #[deprecated(note = "refactored to `email_from_diff` to match upstream")]
```

Comment:
I think it's ok to skip the deprecation here since that's more for the API layer rather than the sys layer